src/main/java/use_case/execute_buy/ExecuteBuyInteractor.java

```diff
@@ -74,19 +73,18 @@ private void updateOrAddStockToPortfolio(Portfolio portfolio, Stock stock, int q
 
     private void handleException(String message) {
```

Comment:
I'm not sure separating a helper method really simplifies the logic, as we still need to pass in a unique string for every error and make sure the string messages on both sides match. I suggest we can simply invoke the `ouputPresenter` in the `catch` statements.

Reply:
I'm not sure whether the session manager implementation will change in the future, but now it's OK to let each presenter handle that. I'll delete the todo.